src/main/java/bio/terra/service/resourcemanagement/ResourceService.java

```diff
@@ -187,7 +188,8 @@ public GoogleBucketResource getOrCreateBucketForBigQueryScratchFile(
         flightId,
         null,
         null,
-        dataset.getProjectResource().getServiceAccount());
+        dataset.getProjectResource().getServiceAccount(),
+        false);
```

Comment:
Might be worth a comment in the method's javadoc that this is always false because autoclass isn't worth the cost given the scratch bucket's usage pattern?